lib/conduit_mqtt.ex

```diff
@@ -24,17 +23,16 @@ defmodule ConduitMQTT do
   end
 
   def start_link(broker, topology, subscribers, opts) do
-    Meta.create(broker)
-    Meta.put_setup_status(broker, :complete) #TODO; put this here for now
+    Meta.create(broker, @pool_size, Enum.count(Map.keys(subscribers)) )
     Supervisor.start_link(__MODULE__, [broker, topology, subscribers, opts], name: name(broker))
   end
 
-  def init([broker, topology, subscribers, opts]) do
+  def init([broker, _topology, subscribers, opts]) do
```

Comment:
We should add a logger warning if topology is not an empty list and then continue normally